packages/ia-topnav/src/more-slider.js

```diff
@@ -1,7 +1,8 @@
-import { html, css } from 'lit-element';
+import { html } from 'lit-element';
 import TrackedElement from './tracked-element';
 import { more as moreMenu } from './data/menus';
 import toSnakeCase from './lib/toSnakeCase';
```

Comment:
QUESTION: In the original merge request, `toSnakeCase.js` seemed like it's 1) converting to camel case, not snake case, and 2) able to get into infinite recursion. I might be wrong, but please do check it out and fix or remove if not used.

Reply:
Yep. Should be `toSentenceCase`. I don't see how there'd be infinite recursion, since it creates an array at space characters and reduces the number of space characters by 1 with each new recursive call.